src/Loader.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import logo from '../src/assets/Ideaoryx.svg'
+import logo from '../src/assets/Ideaoryx.png'
```

Comment:
Is it possible for you to show the output for this for different viewports?
A screenshot would be cool, because I'm not sure if a `png` would scale as well as an `svg` for this case.

Reply:
okay i will change png to svg